### PR TITLE
Fix WiFiNINA getTime()  and getMACAddress()

### PR DESCRIPTION
**Pull request by ysoldak**
Incorporates changes from @kenbell (#249)
Adds an example to verify the fixes.

Verified on Arduino Nano 33 IoT.

This probably shall be merged after #260

### CODE REVIEW

File: examples/wifinina/connect/main.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	println("Gateway IP: ", gateway.String())
+}
+
+func printTime() {
+	print("Time: ")
+	t, err := adaptor.GetTime()
+	if err != nil {
+		println("Unknown (error: ", err.Error(), ")")
+	}
+	fmt.Printf("%v\n", time.Unix(int64(t), 0))
```
Comment (deadprogram) [inline examples/wifinina/connect/main.go]: Tiny nitpick, you need `\r\n` to get the formatting nice.

**Comment by deadprogram** [comment]
Tested and it works as promised! :wrench:

File: examples/wifinina/connect/main.go (hunk 1 above)
Comment (ysoldak) [reply examples/wifinina/connect/main.go]: Sure, will fix later today

**Comment by ysoldak** [comment]
@deadprogram do you think I shall add this example to tests? And since I'm on it also recently added tlsclient?

**Comment by deadprogram** [comment]
Thanks for this @ysoldak now merging.

**Pull request merged**